rs/registry/canister/src/certification.rs

```diff
@@ -19,6 +19,8 @@
 
 #[cfg(target_arch = "wasm32")]
 use dfn_core::api::set_certified_data;
+#[cfg(all(not(target_arch = "wasm32"), test))]
```

Comment:
So this says, print only in testing and non-wasm32 targets, e.g., don't print in production. 

This is unusual, why is this needed here but not in other canisters?

Reply:
Because otherwise, I get an unused import warning/error. The reason for the warning is that the function (`recertify_registry`) that calls `println` is itself compiled conditionally.